src/components/contextual/pages/vebal/LMVoting/LMVoting.vue

```diff
@@ -72,6 +83,70 @@ const hasExpiredLock = computed(
 
 const gaugesTableKey = computed(() => JSON.stringify(isLoading.value));
 
+const gaugesFilteredByExpiring = computed(() => {
+  if (!hideExpiredGauges.value) {
+    return votingGauges.value;
+  }
+
+  return votingGauges.value.filter(gauge => {
+    if (Number(gauge.userVotes) > 0) {
+      return true;
+    }
+    return !expiredGauges.value?.some(expGauge =>
+      isSameAddress(expGauge, gauge.address)
+    );
+  });
+});
+
+const debouncedFilterText = computed({
+  get() {
+    return tokenFilter.value;
+  },
+  set: debounce(newValue => {
+    tokenFilter.value = newValue;
+  }, 500),
+});
```

Comment:
Not a fan of get/set computed properties, it's kind of an anti-pattern, isn't it? Is there an alternative pattern that achieves the same result without computed properties, if so what does it look like?

Reply:
hmm, it seems that we can use `customRef` https://vuejs.org/api/reactivity-advanced.html#customref
I will try

Reply:
yeah, I used `customRef` and it works in the same way